GUI/GUISound.h

```diff
@@ -0,0 +1,265 @@
+#ifndef _GUISOUND_
+#define _GUISOUND_
+
+#include "Sound.h"
+#include "Singleton.h"
+
+#define g_GUISound GUISound::Instance()
+
+namespace RTE {
+
+	/// <summary>
+	/// The singleton loader for all GUI sound effects.
+	/// </summary>
+	class GUISound : public Singleton<GUISound>, public Entity {
+		
+	public:
+
```

Comment:
Why's there this space? We don't have it elsewhere (and I think it looks weird)
Also, you need CLASSINFOGETTERS and ClassInfo definitions for your class.

Reply:
We have this space everywhere.
Macros go directly under public without a space.
Since there aren't any macros here, this is what it is.
Same goes for protected and private, space before members.

Reply:
Ahh, I see the problem. I removed it with TD and TDE cause it's weird looking, but I guess I didn't mention that explicitly.